index.html

```diff
@@ -3,12 +3,46 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="style1.css" rel="stylesheet">
     <link href="style.css" rel="stylesheet">
-    <title></title>
+    <script src="https://use.fontawesome.com/23bea62652.js"></script>
+    <title>Bestselling Author Store Website</title>
 </head>
 <body>
-    <div class="main-container" >
-         
-    </div>
+    <header>
+        <nav class="navbar">
+            <div class="nav__items-container main-container">
+                <div class="logo">
+                    <h2><a target="_self" href="index.html">J. Brown</a></h2>
+                </div>
+                <div class="nav__tabs">
+                <ul>
+                    <li class="nav__item"><a href="#">Home</a></li>
+                    <li class="nav__item"><a href="#">Books</a></li>
+                    <li class="nav__item"><a href="#">In The Press</a></li>
+                    <li class="nav__item"><a href="#">News & Events</a></li>
+                    <li class="nav__item"><a href="#">Contact</a></li>
+                    <li class="nav__item login-icon"><a href="#"><i class="fa fa-user-circle-o" aria-hidden="true"></i> Log in</a></li>
+                </ul>
+                <span class="nav__item cart-icon"><a href="#"><i class="fa fa-shopping-bag" aria-hidden="true"></i></a></span>
+            </div>
+                <div class="bars-icon">
+                    <i class="fa fa-bars" aria-hidden="true"></i>
+                </div>
+            
+            </div>
+        </nav>
+    </header>
+    <section class="banner">
+            <div class="main-container ban__landing-container">
+                    <img src="images/img1.jpg" alt="RealStory" class="ban__landing-img1">
+                    <img src="images/img2.jpg" alt="RealStory" class="ban__landing-img2">
```

Comment:
why do you add two of the images?

Reply:
I cut off parts of the bigger image to fit more with small sizes of the screen